src/NodeApi.Generator/ModuleGenerator.cs

```diff
@@ -38,7 +38,7 @@ public class ModuleGenerator : SourceGenerator, ISourceGenerator
 
     public GeneratorExecutionContext Context { get; protected set; }
 
-    public void Initialize(GeneratorInitializationContext context)
+    public void Initialize(GeneratorInitializationContext _ /*context*/)
```

Comment:
IDK why this suddenly started generating a warning, and why it doesn't repro locally, only on the build machine. It shouldn't warn since the parameter is defined by the interface. [In my PR](https://github.com/microsoft/node-api-dotnet/pull/182/files#diff-cc7d63bc00415ed9a65524baf55cb0a98d2e74cd2c919203a2a17375a5679907) I suppressed with `#pragma` instead`.

Reply:
Please rebase after #185 was merged; it fixes this.